neos/flow-development-collection
# Revert "TASK: Run post-update scripts only once"

This reverts commit d79947442d7de8c59f0d6460e0affef30406148b.

Resolves: #2621

Original by @marcrobertscamao:

This fix resolves the issue of postPackageUpdateAndInstall not installing resources from all installed packages

What I did

Remove the single run code for the postPackageUpdateAndInstall function only

How to verify it

Clone git@github.com:marcrobertscamao/resource-test.git
Run composer install
See resource1.txt in the root and resource2.txt in the root
